src/recipes/pear-manhattan-cocktail.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Banana Smoothie",
```

Comment:
Not sure if this is the right recipe for this file?

Reply:
i push, new fixed on new commit on this